west.yml

```diff
@@ -139,7 +139,11 @@ manifest:
       revision: c39888ff74acf421eeff9a7514fa9b172c3373f7
     - name: trusted-firmware-m
       path: modules/tee/tfm
-      revision: 6edaaaaaa1a227cb1ccc9ef698a9d2e258a9bbd5
+      revision: pull/19/head
+    - name: tfm-mcuboot
+      path: modules/tee/tfm/mcuboot
+      url: https://github.com/mcu-tools/mcuboot.git
```

Comment:
This is not accepted.
All repositories pointed to by Zephyr manifest must be hosted under:
https://github.com/zephyrproject-rtos/

So you should use:
```suggestion
      repo-path: mcuboot
```

Secondly, `west update tfm-mcuboot` fails with:
```
From https://github.com/zephyrproject-rtos/trusted-firmware-m
 * branch              refs/pull/19/head -> FETCH_HEAD
error: The following untracked working tree files would be overwritten by checkout:
        mcuboot/.gitignore
        mcuboot/.gitmodules
        mcuboot/.travis.yml
        mcuboot/LICENSE
...
```
because `mcuboot` is still a folder here:
https://github.com/zephyrproject-rtos/trusted-firmware-m/pull/19

Please coordinate with @nvlsianpu on having the upstream mcuboot tags / SHAs available in Zephyr fork of mcuboot.
Which is actually described:
https://docs.zephyrproject.org/latest/guides/modules.html#allowed-practices
> Upstream changes brought in by performing a Git merge of the intended upstream branch (e.g. main branch, latest release branch, etc.) submitting the result in pull request against the module main branch, and merging the pull request using the Create a merge commit operation. This approach is applicable to modules with an upstream project Git repository. The main advantages of this approach is that the upstream repository history (that is, the original commit SHAs) is preserved in the module repository.

Reply:
Just verified, zephyrproject-rtos/mcuboot#39, does have all the upstream SHAs.

Reply:
Updated. Won't work until zephyrproject-rtos/mcuboot#39 is in.

Reply:
Works now